### ISSUE TITLE
Implement getUserData() and setUserData()

### ISSUE THREAD

**Issue by Tiogaplanet** [adamgreen/MiP_ProMini-Pack#8]
This functionality would be useful for storing variables across power cycles on the MiP.  It could also be used to transfer data between Arduino sketches and Bluetooth apps.

I've started the work, shown in my latest [commit](https://github.com/Tiogaplanet/MiP_ProMini-Pack/commit/9af15418eb187bd17ac02f5edae94ee9a6b7fd4f).  It's not working quite right and I think the problem is in one of the lower transport levels.  My output from ReadWriteEeprom.ino looks like this:

```
ReadWriteEeprom.ino - Writes data to EEPROM and reads it back.
MiP: Bad OOB command byte: 0 (discarded 0 bytes)
MiP: Bad OOB command byte: 0 (discarded 2 bytes)
MiP: Response timeout
Scrambled password: 0
Recovered password: 0
```

Will resume writing/debugging tomorrow.

**Comment by Tiogaplanet** [comment]
OK, I'm really done for tonight now.  I got a little closer to the bullseye with this [commit](https://github.com/Tiogaplanet/MiP_ProMini-Pack/commit/0886da0feea6aaa65d9777bbed81773f051c5148).

My output now looks like this:

```
ReadWriteEeprom.ino - Writes data to EEPROM and reads it back.
Original password: 10
Scrambled password: 255
Recovered password: 32
```

**Comment by adamgreen** [comment]
Implemented as of PR #9
